motion_config.html

```diff
@@ -955,6 +955,12 @@ <h2><a name="Configuration_OptionsAlpha"></a>  Configuration Options-Listed Alph
           <td align="left">stream_auth_method</td>
           <td align="left"><a href="#stream_auth_method" >stream_auth_method</a></td>
         </tr>
+        <tr>
+          <td height="17" align="left"><br /></td>
+          <td align="left">stream_cors_header</td>
```

Comment:
This line(960) and the next one(961) need to be blanks.  They each represent a specific version of Motion.  Since this is new option, it only goes in the last column.